From cd8aca093667635394bed0a98b78d438fb40971f Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Tue, 28 Mar 2017 15:30:40 +0300
Subject: [PATCH 1543/2241] vfio: ahci: cp110: align aHCI compatible for CP110

CP110 aHCI(SATA) unit used the same compatible used for A3700.
After PHY framework introduced to aHCI, CP110 is now using a
deditacted compatible "marvell,armada-cp110-ahci".

This patch aligns VFIO aHCI hook to use the same compatible.

Change-Id: I4b47eb9466890524f1cc91af95630cbbd921ead2
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38098
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/vfio/platform/reset/vfio_platform_ahci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/vfio/platform/reset/vfio_platform_ahci.c b/drivers/vfio/platform/reset/vfio_platform_ahci.c
index ea71543..87629cc 100644
--- a/drivers/vfio/platform/reset/vfio_platform_ahci.c
+++ b/drivers/vfio/platform/reset/vfio_platform_ahci.c
@@ -51,7 +51,7 @@ int vfio_platform_ahci_reset(struct vfio_platform_device *vdev)
 	return 0;
 }
 
-module_vfio_reset_handler("marvell,armada-3700-ahci", vfio_platform_ahci_reset);
+module_vfio_reset_handler("marvell,armada-cp110-ahci", vfio_platform_ahci_reset);
 
 MODULE_VERSION(DRIVER_VERSION);
 MODULE_LICENSE("GPL v2");
-- 
2.7.4

